src/query/functions/fetch.go

```diff
@@ -103,23 +106,32 @@ func (n *FetchNode) fetch(queryCtx *models.QueryContext) (block.Result, error) {
 	sp, ctx := opentracing.StartSpanFromContext(ctx, "fetch")
 	defer sp.Finish()
 
-	timeSpec := n.timespec
-	// No need to adjust start and ends since physical plan
-	// already considers the offset, range
-	startTime := timeSpec.Start
-	endTime := timeSpec.End
+	var (
+		timeSpec = n.timespec
```

Comment:
nit: Should we be consistent and name the field on `FetchNode` to `timeSpec` as well? (to match local var camel casing)